src/stdlib.lisp

```diff
@@ -91,6 +91,11 @@
   (if-not-zero ct
                (remained-constant! X)))
 
+;; perspective constancy constraint
+(defpurefun ((perspective-constancy :@loob) PERSPECTIVE_SELECTOR X)
+            (if-not-zero (* PERSPECTIVE_SELECTOR (prev PERSPECTIVE_SELECTOR))
+                         (remained-constant! X)))
+
```

Comment:
@lorenzogentile404 is the `(perspective-constancy :@loob)` correct ? Shouldn't it be
```
(perspective-constancy  PERSPECTIVE_SELECTOR X)
```
?